spec/fixtures/files/df650pk4327.json

```diff
@@ -0,0 +1 @@
tructuredValue":[],"parallelValue":[],"groupedValue":[],"value":"Stanford University Libraries","type":"topic","identifier":[],"note":[],"appliesTo":[]},{"structuredValue":[],"parallelValue":[],"groupedValue":[],"value":"images","type":"topic","identifier":[],"note":[],"appliesTo":[]},{"structuredValue":[],"parallelValue":[],"groupedValue":[],"value":"interoperability","type":"topic","identifier":[],"note":[],"appliesTo":[]},{"structuredValue":[],"parallelValue":[],"groupedValue":[],"value":"digital library","type":"topic","identifier":[],"note":[],"appliesTo":[]}],"access":{"url":[],"physicalLocation":[],"digitalLocation":[],"accessContact":[{"structuredValue":[],"parallelValue":[],"groupedValue":[],"value":"snydman@stanford.edu","type":"email","identifier":[],"displayLabel":"Contact","note":[],"appliesTo":[]}],"digitalRepository":[],"note":[]},"relatedResource":[{"title":[],"contributor":[],"event":[],"form":[],"language":[],"note":[],"identifier":[],"subject":[],"access":{"url":[{"structuredValue":[],"parallelValue":[],"groupedValue":[],"value":"http://iiif.io","identifier":[],"note":[],"appliesTo":[]}],"physicalLocation":[],"digitalLocation":[],"accessContact":[],"digitalRepository":[],"note":[]},"relatedResource":[]}],"marcEncodedData":[],"adminMetadata":{"contributor":[],"event":[{"structuredValue":[],"type":"creation","date":[{"structuredValue":[],"parallelValue":[],"groupedValue":[],"value":"2015-04-03","encoding":{"code":"edtf","note":[]},"identifier":[],"note":[],"appliesTo":[]}],"contributor":[],"location":[],"identifier":[],"note":[],"parallelEvent":[]}],"language":[],"note":[{"structuredValue":[],"parallelValue":[],"groupedValue":[],"value":"Metadata created by user via Stanford self-deposit application","type":"record origin","identifier":[],"note":[],"appliesTo":[]}],"metadataStandard":[],"identifier":[]},"purl":"https://purl.stanford.edu/df650pk4327"},"identification":{"catalogLinks":[],"sourceId":"hydrus:object-830"},"structural":{"contains":[{"type":"https://cocina.sul.stanford.edu/models/resources/document","externalIdentifier":"https://cocina.sul.stanford.edu/fileSet/df650pk4327-df650pk4327_1","label":"","version":2,"structural":{"contains":[{"type":"https://cocina.sul.stanford.edu/models/file","externalIdentifier":"https://cocina.sul.stanford.edu/file/df650pk4327-df650pk4327_1/2015ARCHIVING_IIIF.pdf","label":"","filename":"2015ARCHIVING_IIIF.pdf","size":1975323,"version":2,"hasMimeType":"application/pdf","sdrGeneratedText":false,"correctedForAccessibility":false,"hasMessageDigests":[{"type":"md5","digest":"23d199406499ab996fae3d4dcf08b21d"},{"type":"sha1","digest":"b27b109dfa6dc983b3bd2836d206292a1ef8d363"}],"access":{"view":"world","download":"world","controlledDigitalLending":false},"administrative":{"publish":true,"sdrPreserve":true,"shelve":true}}]}}],"hasMemberOrders":[],"isMemberOf":["druid:hn730ks3626"]},"created":"2025-07-07T13:17:40.000+00:00","modified":"2025-07-07T13:17:40.000+00:00","lock":"druid:df650pk4327=3=2"}
```

Comment:
Why are we adding these fixtures in particular?  Can we add a manifest and describe their unique characteristics?